conf/test_pep.config

```diff
@@ -0,0 +1,50 @@
+params {
+    config_profile_name        = 'Test PEP profile'
```

Comment:
This shouldn't be a separate test profile, we want this to be used in all existing profiles (as we want to entirely replace the existing system)

Reply:
now I understand :+1:

Reply:
However we need to include the `pep_input_base_dir` now, correct?

Reply:
Do we? I thought that `pep_input_base_dir` will be obtained based on `config.yaml` path

Reply:
Do we? I thought it will be obtained from path to the `config.yaml`